api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -253,3 +288,80 @@ def publish_cdr(cmd_name, args)
   :description => "Publishes a CDR dataset by copying it into a Firecloud CDR project and making it readable by registered users in the corresponding environment",
   :fn => ->(*args) { publish_cdr("publish-cdr", args) }
 })
+
+def create_wgs_extraction_dataset(cmd_name, args)
+  ensure_docker cmd_name, args
+
+  op = WbOptionsParser.new(cmd_name, args)
+
+  op.add_option(
+    "--bq-dataset [dataset]",
+    ->(opts, v) { opts.bq_dataset = v},
+    "BigQuery dataset name for the CDR version (project not included), e.g. " +
+      "'2019Q4R3'. Required."
+  )
+  op.add_option(
+    "--project [project]",
+    ->(opts, v) { opts.project = v},
+    "The Google Cloud project associated with this workbench environment, " +
+      "e.g. all-of-us-rw-staging. Required."
+  )
+  op.opts.tier = "registered"
```

Comment:
default should be controlled, since registered doesn't have genomics (normally); unless that just isn't an option currently, in which case this can wait

Reply:
sure. I've mostly just b/c I've done all my testing so far in the registered tier. Switching the default sounds fine.